lib/chef/knife/ec2_server_create.rb

```diff
@@ -67,10 +67,18 @@ class Ec2ServerCreate < Knife
         :proc => Proc.new { |groups| groups.split(',') }
 
       option :security_group_ids,
-        :short => "-g 'X,Y,Z'",
-        :long => "--security-group-ids 'X,Y,Z'",
-        :description => "The security group ids for this server; required when using VPC,Please provide values in format --security-group-ids 'X,Y,Z'",
-        :proc => Proc.new { |security_group_ids| security_group_ids.split(',') }
+        :short => "-g SECURITY_GROUP_IDS",
+        :long => "--security-group-ids",
```

Comment:
@mwrock @btm has suggested to raise an error for option --security-group-ids and show user message as deprecated and use new option --security-group-id. I am making changes in code so please let me know what is the final approach that we should take.